server/routes/html.js

```diff
@@ -48,7 +49,7 @@ export default () => {
       AUTH0_CLIENT_ID: config('EXTENSION_CLIENT_ID'),
       BASE_URL: urlHelpers.getBaseUrl(req),
       BASE_PATH: urlHelpers.getBasePath(req),
-      TITLE: config('TITLE')
+      TITLE: (req.custom_style && req.custom_style.title) ? req.custom_style.title : config('TITLE')
```

Comment:
Remove all changes in this file. All of this logic should happen on the client side after you logged in.